projects/1.16/assets/goblins-dungeons/goblinsanddungeons/lang/zh_cn.json

```
@@ -0,0 +1,130 @@
+﻿{
+  "itemGroup.goblinsanddungeons_item_group": "哥布林与地下城",
+  "item.goblinsanddungeons.goblin_crown": "哥布林王冠",
+  "item.goblinsanddungeons.gob_spawn_egg": "哥布林战士刷怪蛋",
+  "item.goblinsanddungeons.hobgob_spawn_egg": "大型哥布林刷怪蛋",
+  "item.goblinsanddungeons.goblo_spawn_egg": "恶鬼哥布林刷怪蛋",
+  "item.goblinsanddungeons.garch_spawn_egg": "哥布林射手刷怪蛋",
+  "item.goblinsanddungeons.goom_spawn_egg": "烟雾弹哥布林刷怪蛋",
+  "item.goblinsanddungeons.mimic_spawn_egg": "宝箱怪刷怪蛋",
+  "item.goblinsanddungeons.gobber_spawn_egg": "哥布林守卫刷怪蛋",
+  "item.goblinsanddungeons.ogre_spawn_egg": "哥布林巨魔刷怪蛋",
+  "item.goblinsanddungeons.goblin_king_spawn_egg": "哥布林王刷怪蛋",
+  "item.goblinsanddungeons.goblin_soul_bullet": "灵魂子弹",
+  "item.goblinsanddungeons.staff_amethyst": "法杖",
+  "block.goblinsanddungeons.urn": "陶罐",
+  "block.goblinsanddungeons.scoria": "火山岩",
+  "block.goblinsanddungeons.scoria_bricks": "火山岩砖块",
+  "block.goblinsanddungeons.cracked_scoria_bricks": "裂纹火山岩砖块",

```

```suggestion
  "block.goblinsanddungeons.scoria_bricks": "火山岩砖",
  "block.goblinsanddungeons.cracked_scoria_bricks": "裂纹火山岩砖",
```
原版没有对应砖物品的Bricks方块应该是统一不用`块`的